crates/fmt/testdata/VariableAssignment/bracket-spacing.fmt.sol

```diff
@@ -8,8 +8,9 @@ contract TestContract {
         (, uint256 second) = (1, 2);
         (uint256 listItem001) = 1;
         (uint256 listItem002, uint256 listItem003) = (10, 20);
-        (uint256 listItem004, uint256 listItem005, uint256 listItem006) =
-            (10, 20, 30);
+        (uint256 listItem004, uint256 listItem005, uint256 listItem006) = (
+            10, 20, 30
+        );
```

Comment:
hmm is this right? shouldn't each the tuple element go on a new line? I don't mind it, just checking